Assets/HoloToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs

```diff
@@ -94,24 +95,37 @@ private void MaybeInitAsServer()
                 return;
             }
 
+            StartCoroutine(InitAsServer());
+        }
+
+        private IEnumerator InitAsServer()
+        {
             Debug.Log("Acting as host");
             // StopBroadcast will also 'StopListening'
             StopBroadcast();
 
+            // Work-around when building to the HoloLens with "Compile with .NET Native tool chain".
+            // Need a frame of delay after StopBroadcast() otherwise clients won't connect.
+            yield return null;
```

Comment:
Great question! I wasn't able to find anything exposed in the APIs to "yield return WaitUntil". I'm reaching out to get some eyes at Unity to check in on this as well.

I was originally waiting 2 seconds, but a single frame was producing successful discoverability too. It's extremely reproducible in connection issues without this. Before it seemed about 1 out of 10 times clients weren't finding a server with tool chain enabled.